styles/style.css

```diff
@@ -1,31 +1,89 @@
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
 h1 {
   text-align: center;
+  margin-top: 120px;
+  font-family: roboto;
+  color: #383838;
 }
 hr {
   border: 0;
   background-color: #f5bd41;
   width: 100px;
   height: 4px;
 }
-/* TEXT */
-.text {
+.features {
   display: flex;
-  justify-content: center;
-  margin-top: 20px;
-  margin-right: 40px;
+  margin: 50px 0;
 }
-.text_item {
+.features_item {
   width: 20%;
+  text-align: center;
```

Comment:
It's too different, please fix
<img width="1289" alt="Screenshot 2022-03-11 at 22 07 43" src="https://user-images.githubusercontent.com/25488411/157936109-8e47a0a2-b8d9-45ac-a3ef-0d569cca7f3d.png">